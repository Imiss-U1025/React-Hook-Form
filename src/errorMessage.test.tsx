import * as React from 'react';
import { render } from '@testing-library/react';
import { ErrorMessage } from './index';

describe('React Hook Form Error Message', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <ErrorMessage
        errors={{ test: { type: 'test', message: 'test' } }}
        as={<span />}
        name="test"
      />,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
