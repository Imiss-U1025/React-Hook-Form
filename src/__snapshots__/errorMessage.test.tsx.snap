// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Hook Form Error Message should render correctly 1`] = `
<DocumentFragment>
  <span>
    test
  </span>
</DocumentFragment>
`;
